glustercli/cmd/volume.go

```diff
@@ -67,8 +66,6 @@ func init() {
 	volumeCmd.AddCommand(volumeDeleteCmd)
 
 	volumeCmd.AddCommand(volumeGetCmd)
-	volumeCmd.AddCommand(volumeResetCmd)
-	volumeCmd.AddCommand(volumeSizeCmd)
```

Comment:
This is removed intentionally?

Reply:
No, I am really sorry, seems like I removed the volumeSizeCmd  by mistake. volumeResetCmd was removed because I wanted it to be seperate just like volumeSetCmd.

Reply:
please add it back

Reply:
Will do